lms/static/js/dialog_tab_controls.js

```diff
@@ -0,0 +1,77 @@
+var DialogTabControls = (function() {
+    'use strict';
+
+    var focusableChildren,
+        numElements,
+        currentIndex,
+        setCurrentIndex = function(currentElement) {
```

Comment:
I don't understand why this is necessary. What happens if you don't muck with the focusable children and just allow keyboard focus to work naturally? I've never had to override the default focus order before.

FYI @cptvitamin